content/project/conduct.adoc

```diff
@@ -7,180 +7,246 @@ section: conduct
-on link:http://lists.jenkins-ci.org/mailman/listinfo[lists.jenkins-ci.org].
-
-==== IRC Channels
+All link:/mailing-lists[mailing lists] hosted on Google Groups and other platforms.
 
-Documented in the
-link:/chat[IRC Channel] wiki
-page, e.g.
+==== Chats
 
-* `#jenkins`
-* `#jenkins-meeting`
-* `#jenkins-infra`
-* `#jenkins-community`
+* All chats documented on the link:/chat[this page], e.g. IRC channels: `#jenkins`, `#jenkins-meeting`, `#jenkins-infra`, etc.
+* Gitter channels within the link:https://gitter.im/jenkinsci/home[https://gitter.im/jenkinsci/] space
+* Jenkins chats hosted by Linux Foundation and its subsidiaries including Continuous Delivery Foundation 
 
 ==== Events
 
-* link:/projects/jam/[Jenkins
-  Area Meetup]
+* link:/projects/jam/[Jenkins Area Meetups]
+* link:/events/online-meetup[Jenkins Online Meetup]
 * Events where Jenkins is officially being represented (e.g.
   link:https://fosdem.org[FOSDEM], link:https://socallinuxexpo.org/[SCaLE])
-* Other Jenkins groups (such as conferences, meetups, and other unofficial
-  forums) are encouraged to adopt this Code of Conduct. Those groups must
-  provide their own moderators and/or reporting system.
 
+=== Other Jenkins Communities
+
+Other Jenkins groups (such as conferences, meetups, and other unofficial forums) and local communities are encouraged to adopt this Code of Conduct.
+Those groups must provide their own moderators and/or reporting system.
 
 == Reporting
 
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the link:/project/board[Jenkins board].
 If you feel somebody has breached this code, please send an email with the
-relevant information (links, etc) to jenkinsci-board@googlegroups.com. This
-email list is only readable by the
-link:/project/board[Governance
-Board] members.
-
-Please understand that the board may not have all of the necessary context and
-history, so it's better to be more verbose than less verbose about such
-matters.
+relevant information (links, etc) to `jenkinsci-board@googlegroups.com`.
+This email list is only readable by the link:/project/board[Governance Board] members.
+The board may not have all of the necessary context and history,
+so it's better to describe the issue thoroughly.
+All complaints will be reviewed
+and investigated and will result in a response that is deemed necessary and
+appropriate to the circumstances.
 
-If you believe one of the members of the board has violated the code of conduct
+If you believe one of the board members has violated the code of conduct
 above, please email one of the other members of the Governance Board with the
 details (their emails are visible on the
 link:/project/board[Governance
 Board] page).
 
+If the desired resolution cannot be reached on the Jenkins community level,
+an issue can be escalated to the Continuous Delivery Foundation (CDF) by contacting the project team at `conduct@cd.foundation`.
```

Comment:
Same question about a mailto link, since CDF uses a mailto link in their markdown.
```suggestion
an issue can be escalated to the Continuous Delivery Foundation (CDF) by contacting the project team at mailto:conduct@cd.foundation[`conduct@cd.foundation`].
```